eeglabio/epochs.py

```diff
@@ -7,7 +7,7 @@
 
 def export_set(fname, data, sfreq, events, tmin, tmax, ch_names, event_id=None,
                ch_locs=None, annotations=None, ref_channels="common",
-               precision="single"):
+               precision="single", epoch_indices=None):
```

Comment:
Let's add a * somewhere, at the very least
```suggestion
               precision="single", *, epoch_indices=None):
```